naga/Cargo.toml

```diff
@@ -73,16 +81,25 @@ hlsl-out-if-target-windows = []
 
 compact = []
 
+## Provide rich error messages using codespan-reporting
+codespan-reporting = ["std", "dep:codespan-reporting", "dep:termcolor"]
```

Comment:
I worry that cordoning codespan-reporting off completely will mean that `wgpu-core` won't be able to format errors like it expects to, as it definitely pretty prints the errors there.